coverage_point_calculator/src/location.rs

```diff
@@ -17,20 +11,42 @@ pub struct LocationTrust {
     pub trust_score: Decimal,
 }
 
-pub(crate) fn clean_trust_scores(
-    trust_scores: Vec<LocationTrust>,
-    ranked_coverage: &[RankedCoverage],
-) -> Vec<LocationTrust> {
-    let any_boosted_hexes = ranked_coverage.iter().any(|hex| hex.boosted.is_some());
-
-    if any_boosted_hexes {
-        trust_scores
-            .into_iter()
-            .map(LocationTrust::into_boosted)
-            .collect()
-    } else {
-        trust_scores
+/// Returns the trust multiplier for a given radio type and distance to it's asserted location.
+///
+/// [HIP-119: Gaming Loopholes][gaming-loopholes]
+///
+/// [gaming-loopholes]: https://github.com/helium/HIP/blob/main/0119-closing-gaming-loopholes-within-the-mobile-network.md#maximum-asserted-distance-difference
+pub fn asserted_distance_to_trust_multiplier(
+    radio_type: RadioType,
+    meters_to_asserted: Meters,
+) -> Decimal {
+    match radio_type {
+        RadioType::IndoorWifi | RadioType::IndoorCbrs => match meters_to_asserted {
```

Comment:
my reading of hip119 is that the trust score change applies to wifi radios only ( see phase 1 of implementation phases ). Has there been confirmation these trust scores based on specified distances should be applied to cbrs radios?

Reply:
Indeed it does say that. 
CBRS radios are intercepted earlier when validating heartbeats, but I've updated this function to return `dec!(1.0)` for CBRS if somehow they make it here.
[c3e8194](https://github.com/helium/oracles/pull/840/commits/c3e81944b6d3b1b82253ce426551d794a74a5e7d)

Reply:
yeah the change also helps to convey the intent better to the reader as well as aligning better with the hip.  better for long term understanding